projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -91,6 +91,30 @@ export async function jobRequest(endPoint: string): Promise<void> {
   }
 }
 
+export async function jobTransformRequest(): Promise<{
+  message: String;
+  status: boolean;
+}> {
+  if (
+    (await checkIfVdkOptionDataIsDefined(VdkOption.NAME)) &&
+    (await checkIfVdkOptionDataIsDefined(VdkOption.TEAM)) &&
+    (await checkIfVdkOptionDataIsDefined(VdkOption.PATH))
+  ) {
+    try {
+      const data = await requestAPI<serverVdkOperationResult>('transform', {
+        body: JSON.stringify(getJobDataJsonObject()),
+        method: 'POST'
+      });
+      return { message: data['message'], status: data['message'] == '0' };
+    } catch (error) {
+      showError(error);
+      return { message: '', status: false };
+    }
+  } else {
+    return { message: '', status: false };
```

Comment:
are you going to improve this error message?

Reply:
@duyguHsnHsn maybe we should improve the error messages for all the operations in general? Should we create a follow-up story and take this there?

Reply:
@yonitoo yes, please create an issue for that

Reply:
Added an issue for it - https://github.com/vmware/versatile-data-kit/issues/2333.